src/types/NavigationLocations.ts

```diff
@@ -0,0 +1,7 @@
+type NavigationLocations =
```

Comment:
Since a value of this type represents a single location that is being viewed, I wonder if we want to use the singular name `NavigationLocation` here instead?